cms/templates/cms/program_page.html

```diff
@@ -58,7 +58,7 @@ <h1 class="title">
         <div class="description">
           {{ page.title_over_image|richtext }}
         </div>
-        {% if courses %}
+        {% if not page.program_subscribe_link %}
```

Comment:
This logic is pretty confusing. I know you're trying to make the minimal change, but would you mind refactoring it to put `if page.program_subscribe_link` first?

Reply:
It is actually first here. I am removing the courses check since it is redundant now.

Reply:
I see what you mean :)